detekt.yml

```diff
@@ -1177,9 +1179,15 @@ datadog:
       - "kotlin.collections.MutableSet.map(kotlin.Function1)"
       - "kotlin.collections.MutableSet.remove(java.io.File)"
       - "kotlin.collections.MutableSet.remove(com.datadog.android.v2.core.internal.storage.ConsentAwareStorage.Batch)"
+      - "kotlin.collections.Iterable.toMap(kotlin.collections.MutableMap)"
```

Comment:
minor: should go up, to L1058 (where `Iterable` is declared)

Reply:
Actually that's a duplicate from a rebase :)